src/cljs/omnext_to_datomic/core.cljs

```diff
@@ -0,0 +1,288 @@
+(ns omnext-to-datomic.core
+  (:require
+   [cognitect.transit :as transit]
+   [goog.dom :as gdom]
+   [om.next :as om :refer-macros [defui]]
+   [sablono.core :as html :refer-macros [html]]
+   [ajax.core :refer [GET POST]]))
+
+(enable-console-print!)
+
+
+(def init-data {:form/author ""
+                :nav/index 0
+                :comment/list [] })
+
+;; -----------------------------------------------------------------------------
+;; Ajax
+
+
+(defn merge-response [state res]
+  (println res)
+  (let [comments (:comment/list res)
+        by-time  (into {}
+                       (map (fn [c][(:comment/time c) c]) comments))
+        comments (map (fn [c] [:comment/by-time (:comment/time c)]) comments)]
+    (-> state
+        (update :comment/by-time merge by-time)
+        (update :comment/list concat comments))))
+
+(defn error-handler [{:keys [status status-text]}]
+  (.log js/console (str "something bad happened: " status " " status-text)))
```

Comment:
Unless you had a specific reason for this, since you invoked `enable-console-print!` this line could be made into `println` like the others and still have the text print via `console.log`